stader-cli/node/send-el-rewards.go

```diff
@@ -2,6 +2,7 @@ package node
 
 import (
 	"fmt"
+
```

Comment:
let's see if we can avoid getting these changes to the PR. creates unnecessary diff. 

I think it might be part of a lint tho